From ltsi-dev-bounces@lists.linuxfoundation.org Mon May 28 23:23:31 2012
From: "Shimoda, Yoshihiro" <yoshihiro.shimoda.uh@renesas.com>
Date: Tue, 29 May 2012 15:22:59 +0900
Subject: [LTSI-dev] [PATCH 06/12] USB: irq: Remove IRQF_DISABLED
To: ltsi-dev@lists.linuxfoundation.org
Message-ID: <4FC46B43.3050307@renesas.com>


>From 1c1d88148ed0c54c58f7426b49c417c44bb68eeb Mon Sep 17 00:00:00 2001
From: Yong Zhang <yong.zhang0@gmail.com>
Date: Wed, 7 Sep 2011 16:10:52 +0800
Subject: USB: irq: Remove IRQF_DISABLED

This flag is a NOOP and can be removed now.

Signed-off-by: Yong Zhang <yong.zhang0@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
(cherry picked from commit b5dd18d8747010e3f3eb1cc76a49f94291938559)

Conflicts:

    drivers/usb/core/hcd-pci.c
    drivers/usb/gadget/at91_udc.c
    drivers/usb/gadget/fusb300_udc.c
    drivers/usb/gadget/imx_udc.c
    drivers/usb/gadget/m66592-udc.c
    drivers/usb/gadget/mv_udc_core.c
    drivers/usb/gadget/omap_udc.c
    drivers/usb/gadget/pxa25x_udc.c
    drivers/usb/gadget/r8a66597-udc.c
    drivers/usb/gadget/s3c2410_udc.c
    drivers/usb/host/ehci-ath79.c
    drivers/usb/host/ehci-au1xxx.c
    drivers/usb/host/ehci-fsl.c
    drivers/usb/host/ehci-mxc.c
    drivers/usb/host/ehci-octeon.c
    drivers/usb/host/ehci-omap.c
    drivers/usb/host/ehci-orion.c
    drivers/usb/host/ehci-ps3.c
    drivers/usb/host/ehci-s5p.c
    drivers/usb/host/ehci-sh.c
    drivers/usb/host/ehci-spear.c
    drivers/usb/host/ehci-tegra.c
    drivers/usb/host/ehci-vt8500.c
    drivers/usb/host/fhci-hcd.c
    drivers/usb/host/imx21-hcd.c
    drivers/usb/host/isp116x-hcd.c
    drivers/usb/host/isp1362-hcd.c
    drivers/usb/host/isp1760-if.c
    drivers/usb/host/ohci-ath79.c
    drivers/usb/host/ohci-au1xxx.c
    drivers/usb/host/ohci-da8xx.c
    drivers/usb/host/ohci-ep93xx.c
    drivers/usb/host/ohci-octeon.c
    drivers/usb/host/ohci-omap.c
    drivers/usb/host/ohci-omap3.c
    drivers/usb/host/ohci-pnx4008.c
    drivers/usb/host/ohci-pnx8550.c
    drivers/usb/host/ohci-ppc-of.c
    drivers/usb/host/ohci-ppc-soc.c
    drivers/usb/host/ohci-ps3.c
    drivers/usb/host/ohci-pxa27x.c
    drivers/usb/host/ohci-s3c2410.c
    drivers/usb/host/ohci-sa1111.c
    drivers/usb/host/ohci-sh.c
    drivers/usb/host/ohci-sm501.c
    drivers/usb/host/ohci-spear.c
    drivers/usb/host/ohci-ssb.c
    drivers/usb/host/ohci-tmio.c
    drivers/usb/host/r8a66597-hcd.c
    drivers/usb/host/sl811-hcd.c
    drivers/usb/host/xhci-pci.c
    drivers/usb/musb/musbhsdma.c
    drivers/usb/otg/isp1301_omap.c
    drivers/usb/renesas_usbhs/mod.c

Signed-off-by: Yoshihiro Shimoda <yoshihiro.shimoda.uh@renesas.com>
---
 drivers/usb/gadget/r8a66597-udc.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/usb/gadget/r8a66597-udc.c b/drivers/usb/gadget/r8a66597-udc.c
index fe3313a..460b3bf 100644
--- a/drivers/usb/gadget/r8a66597-udc.c
+++ b/drivers/usb/gadget/r8a66597-udc.c
@@ -1618,7 +1618,7 @@ static int __init r8a66597_probe(struct platform_device *pdev)

 	disable_controller(r8a66597); /* make sure controller is disabled */

-	ret = request_irq(irq, r8a66597_irq, IRQF_DISABLED | IRQF_SHARED,
+	ret = request_irq(irq, r8a66597_irq, IRQF_SHARED,
 			udc_name, r8a66597);
 	if (ret < 0) {
 		printk(KERN_ERR "request_irq error (%d)\n", ret);
-- 
1.7.1

_______________________________________________
LTSI-dev mailing list
LTSI-dev@lists.linuxfoundation.org
https://lists.linuxfoundation.org/mailman/listinfo/ltsi-dev

